create.sh

```diff
@@ -76,6 +76,11 @@ az group deployment create -g $RESOURCEGROUP -n azuredeploy --template-file _dat
 
 KUBECONFIG=aks/admin.kubeconfig helm install --namespace $RESOURCEGROUP pkg/helm/chart -f _data/_out/values.yaml -n $RESOURCEGROUP >/dev/null
 
+# Configure dev settings
```

Comment:
- bring this above helm install - if dev.sh has to create the namespace to do so, that's fine
- delete the comment or make it meaningful
- rename dev.sh to something meaningful